src/components/DepartmentsList/DepartmentsList.tsx

```diff
@@ -14,7 +14,7 @@ interface DepartmentsListProps {
 
 const DepartmnetsList: React.FC<DepartmentsListProps> = ({ departments }) => {
   if (departments.length === 0) {
-    return <NotFound />;
+    return <NotFound errMsg=""/>;
```

Comment:
1. what for? :)
2. chng t errorMessage  pls

Reply:
there's no need actually I did just to render error message it the NotFound component for now

Reply:
you can make errorMessage optional param